apps/src/codebridge/Codebridge.tsx

```diff
@@ -94,6 +96,14 @@ export const Codebridge = React.memo(
     } else {
       throw new Error('Cannot render codebridge - no layout provided');
     }
+    // gridLayout is a css string the defines the components in the grid layout.
+    // In order to find which components are in the grid layout, we remove all quotes
+    // from the string and tokenize it.
+    const gridLayoutKeys = gridLayout
+      .trim()
+      .replaceAll(`"`, '')
+      .split(' ')
+      .map(key => key.trim());
```

Comment:
Is this a bit hacky?  It seems there could be a common source of truth that can generate the grid string and also be used here.

Reply:
It is a bit hacky, but it was hackier [before](https://github.com/code-dot-org/code-dot-org/blob/b128e412e16188bc063f61263ed8ac78331ca06c/apps/src/codebridge/Codebridge.tsx#L120) (and broke with my change because I added a new key that contained other keys (`workspace` and `console`). I think this should work for our current purposes, since the grid layout string is always the same format, and we define it in code. 

Long-term we want to completely move away from grid layout so I didn't want to overcomplicate things with some sort of generator.